vent/menus/main.py

```diff
@@ -350,6 +354,14 @@ def create(self):
                                  labelColor='DEFAULT', value=Cpu())
         self.gpufield = self.add(npyscreen.TitleFixedText, name='GPUs:',
                                  labelColor='DEFAULT', value=Gpu()[1])
+        self.location = self.add(npyscreen.TitleFixedText,
+                                 name='.vent:',
+                                 value=PathDirs().meta_dir,
+                                 labelColor='DEFAULT')
+        self.file_drop = self.add(npyscreen.TitleFixedText,
+                                  name='File Drop:',
+                                  value=template.section("main")[1][0][1],
```

Comment:
I know that will return a tuple of `(bool, str)`. Maybe just display the `str`? 
So the vent menu would show `File Drop: [(files, /tmp/vent_files)]`
Is that fine?